singleton-ruby.gemspec

```diff
@@ -17,21 +17,15 @@ Gem::Specification.new do |s|
   s.test_files    = s.files.grep(%r{^(test|spec|features)/})
   s.require_paths = ["lib"]
 
-  s.add_development_dependency('multi_json', '~> 1.0')
   s.add_development_dependency('request_store', '~> 1.1')
-  s.add_development_dependency('twitter_cldr', '~> 6.6')
   s.add_development_dependency('webmock', '~> 2.0')
   s.add_development_dependency('rspec', '~> 3.0')
   s.add_development_dependency('rdoc', '>= 2.4.2', '< 6.0')
-  s.add_development_dependency('rubocop', '~> 0.49')
 
-  s.add_dependency('http-accept', '>= 1.7.0', '< 2.0')
-  s.add_dependency('http-cookie', '>= 1.0.2', '< 2.0')
-  s.add_dependency('mime-types', '>= 1.16', '< 4.0')
-  s.add_dependency('netrc', '~> 0.8')
   s.add_dependency('multi_json', '~> 1.0')
-  s.add_dependency('rest-client', '~> 2.0')
   s.add_dependency('twitter_cldr', '~> 6.6')
+  s.add_dependency('faraday')
+  s.add_dependency('faraday_middleware')
```

Comment:
> I had to explicitly add faraday in my sample app's Gemspec. Otherwise, this causes an error. Please fix.

Gemspec is for a Gem, not an app. Use Gemfile instead.

Reply:
I meant Gemfile, not Gemspec. I can show you.

Reply:
I tested again just now and I didn't get the error. Not sure what changed.